File: utilities/src/main/kotlin/lunalauf/rms/utilities/network/communication/message/response/MinigameRecordFailedResponse.java
package lunalauf.rms.utilities.network.communication.message.response;

import com.google.gson.annotations.SerializedName;
import lunalauf.rms.utilities.network.communication.message.type.ResponseType;

public class MinigameRecordFailedResponse extends MinigameRecordResponse {

    @SerializedName("msg")
    public String causeMessage;

    public MinigameRecordFailedResponse() {
        super(ResponseType.MINIGAME_RECORD_FAILED);
    }

}

File: utilities/src/main/kotlin/lunalauf/rms/utilities/network/communication/message/response/MinigameRecordFailedResponse.kt
package lunalauf.rms.utilities.network.communication.message.response

import com.google.gson.annotations.SerializedName
import lunalauf.rms.utilities.network.communication.message.type.ResponseType

class MinigameRecordFailedResponse : MinigameRecordResponse(ResponseType.MINIGAME_RECORD_FAILED) {
    @SerializedName("msg")
    var causeMessage: String? = null
}
